mava/components/jax/training/losses.py

```diff
@@ -58,7 +65,20 @@ def loss_grad_fn(
             advantages: Dict[str, jnp.ndarray],
             behavior_values: Dict[str, jnp.ndarray],
         ) -> Tuple[Dict[str, jnp.ndarray], Dict[str, Dict[str, jnp.ndarray]]]:
-            """Surrogate loss using clipped probability ratios."""
+            """Surrogate loss using clipped probability ratios.
+
+            Args:
+                params: network parameters.
+                observations: agent observations.
+                actions: actions the agents took.
+                behaviour_log_probs: TODO(Matthew): not sure what this is.
```

Comment:
@mmorris44 It's the log probability of the action taken at that timestep. `behavior_values` on line 77 is the estimated value from the critic at that timestep if I'm not mistaken? The name `behavior_values` might be a bit misleading tbh.

Reply:
Thanks for the clarifications! I've added the info.